[ 2024-04-24 14:43:31,496 ] 91 main_controller - INFO - Subprocess 0 started with trial number 0
[ 2024-04-24 14:43:31,496 ] 91 main_controller - ERROR - Account is a dictionary, convert to Account object: Username: luthien5921@gmail.com, Password: aichi@5921
[ 2024-04-24 14:43:32,653 ] 91 main_controller - INFO - Browser opened
[ 2024-04-24 14:43:42,727 ] 91 main_controller - INFO - Logged in successfully
[ 2024-04-24 14:43:42,928 ] 91 main_controller - INFO - Screenshot saved to C:\Users\USER\Projects\AichiTool\AichiSelenium\logs\2024_04_24\14_43_31\screenshot_luthien5921.png
[ 2024-04-24 14:43:42,929 ] 91 main_controller - INFO - Finding forms
[ 2024-04-24 14:43:51,217 ] 79 main_controller - ERROR - Caught exception: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.61)
Stacktrace:
	GetHandleVerifier [0x00007FF61B9DF922+3045234]
	(No symbol) [0x00007FF61B683652]
	(No symbol) [0x00007FF61B547564]
	(No symbol) [0x00007FF61B51DB4F]
	(No symbol) [0x00007FF61B5C1087]
	(No symbol) [0x00007FF61B5D8871]
	(No symbol) [0x00007FF61B5B9B53]
	(No symbol) [0x00007FF61B58813C]
	(No symbol) [0x00007FF61B588D71]
	GetHandleVerifier [0x00007FF61BA0E89D+3237613]
	GetHandleVerifier [0x00007FF61BA50BA7+3508727]
	GetHandleVerifier [0x00007FF61BA4976F+3478975]
	GetHandleVerifier [0x00007FF61B7BF1C6+815126]
	(No symbol) [0x00007FF61B68ED0F]
	(No symbol) [0x00007FF61B689854]
	(No symbol) [0x00007FF61B6899E2]
	(No symbol) [0x00007FF61B67A574]
	BaseThreadInitThunk [0x00007FF92E12257D+29]
	RtlUserThreadStart [0x00007FF93022AA48+40]
Traceback (most recent call last):
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\main_controller.py", line 72, in subprocess
    list_form_info = finder.find(browser)
  File "C:\Users\USER\Projects\AichiTool\AichiSelenium\src\controllers\finder.py", line 30, in find
    browser.refresh()
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 570, in refresh
    self.execute(Command.REFRESH)
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Users\USER\anaconda3\envs\aichi\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.61)
Stacktrace:
	GetHandleVerifier [0x00007FF61B9DF922+3045234]
	(No symbol) [0x00007FF61B683652]
	(No symbol) [0x00007FF61B547564]
	(No symbol) [0x00007FF61B51DB4F]
	(No symbol) [0x00007FF61B5C1087]
	(No symbol) [0x00007FF61B5D8871]
	(No symbol) [0x00007FF61B5B9B53]
	(No symbol) [0x00007FF61B58813C]
	(No symbol) [0x00007FF61B588D71]
	GetHandleVerifier [0x00007FF61BA0E89D+3237613]
	GetHandleVerifier [0x00007FF61BA50BA7+3508727]
	GetHandleVerifier [0x00007FF61BA4976F+3478975]
	GetHandleVerifier [0x00007FF61B7BF1C6+815126]
	(No symbol) [0x00007FF61B68ED0F]
	(No symbol) [0x00007FF61B689854]
	(No symbol) [0x00007FF61B6899E2]
	(No symbol) [0x00007FF61B67A574]
	BaseThreadInitThunk [0x00007FF92E12257D+29]
	RtlUserThreadStart [0x00007FF93022AA48+40]

